vendor/brave-ios/Ads/Generated/NativeAdsClient.h

```diff
@@ -28,6 +28,7 @@ class NativeAdsClient : public ads::AdsClient {
   void Save(const std::string & name, const std::string & value, ads::ResultCallback callback) override;
   void Load(const std::string & name, ads::LoadCallback callback) override;
   void LoadUserModelForId(const std::string & id, ads::LoadCallback callback) override;
+  void SearchBrowsingHistory(const int max_count, const int days_ago, ads::SearchBrowsingHistoryCallback callback) override;
```

Comment:
Wrong method name based on what I've seen in `ads_client.h`, should be `GetBrowsingHistory` and `ads::GetBrowsingHistoryCallback`

Also, this is not the only piece that needs to be implemented for iOS, missing pieces:

- Implementation in `NativeAdsClient.mm`
- Bridge method in `NativeAdsClientBridge.h`
- Implementation of bridge method in `BATBraveAds.mm`

Is this feature supposed to function in iOS? Doc in AdsClient does not explain what the strings are? Are they a list of URLs? 

If it does have to function in iOS, what is the target date?

Reply:
ah good point, this might not ship for iOS initially. I'll get back to you on this and will remove or augment depending on scope @kylehickinson

Reply:
@kylehickinson iOS is not part of this ticket, I removed the changes

Reply:
@moritzhaller Unfortunately we still need to implement the AdsClient methods or we will fail to compile, but at the end of it the `GetBrowsingHistory` can be a function that just immediately calls back with an empty list always. If you need me to add the code to your PR here just let me know

Reply:
thanks @kylehickinson, I've tried to add the stubs. Could you have a look, I'm pretty sure the callback, not sure how to best initialise an empty array :)